server/routes/prisonerList/prisonerListController.test.ts

```diff
@@ -72,14 +72,7 @@ describe('prisonerListController', () => {
         statusFilter: '',
         sortBy: 'reception-date',
         sortOrder: 'descending',
-        items: [
-          {
-            href: '?sort=reception-date,descending&page=1',
-            selected: true,
-            text: '1',
-            type: undefined,
-          },
-        ],
+        items: [],
```

Comment:
I made a change to the View class (👇 ) so that if there is only 1 record in the items array (and therefore it must be page 1 / current page), then return an empty array instead.
This was to prevent a lone `1` being displayed as the pagination controls, and instead there are no pagination controls and just the record count (see screenshot in main description)